source/agora/consensus/state/UTXOSet.d

```diff
@@ -137,18 +117,83 @@ public class UTXOSet
 
     ***************************************************************************/
 
-    public UTXOFinder getUTXOFinder () @trusted nothrow
+    public abstract UTXOFinder getUTXOFinder () nothrow @trusted;
```

Comment:
Why abstract ? The implementation is the same, isn't it ?

Reply:
Yes. I tried to implement it in the base class and making `findUTXO` abstract, but it caused too many errors. Probably due to the delegate and alias.
Currently, `UTXOSet` uses `findUTXO` and `TestUTXOSet` uses `findUTXO_`. They seemed to be overloadable but  required code changes in other parts.